src/ReferenceFEs/StrangQuadratures.jl

```diff
@@ -24,7 +24,7 @@ end
 
 const _strang_tri_k2n = Dict(1=>1,2=>3,3=>4,4=>6,5=>7,7=>13,9=>19,11=>28)
 
-function _strang_quad_tri(degree)
+function _strang_quad_tri(degree;T::Type{<:AbstractFloat}=Float64)
```

Comment:
Why do you use `T` instead of `fptype`? I would try to stick to the same name for consistency reasons. 

On the other hand, I think `T` is a quite standard name for eltypes in Gridap and Julia. Not sure about `fptype`. What do you think?

Reply:
yes sure I can continue to use `fptype` instead of `T`, in other internal functions I had used `T` because `T` was already being used, for example, `TensorProductQuadratures`.

Yes I thought about it, but the problem is as a keyword argument `T` doesn't convey much information to the user.

Reply:
> Yes I thought about it, but the problem is as a keyword argument T doesn't convey much information to the user.

I don't agree here. To a reasonably experienced Julia programmer, `T` tells a lot. 

`T` is the canonical name for the `eltype` type parameter in `Array{T,N}` and in many other types. 

If instead of `T` we called it, say, `H`, I would agree with you.

Reply:
True.., I can make it `T` everywhere if that is better?

Reply:
sure will do it now, along with any other changes, thank you for the decision..

Reply:
Done!